src/garage/torch/algos/sac.py

```diff
@@ -69,6 +69,10 @@ class SAC(OffPolicyRLAlgorithm):
         optimizer(torch.optim): optimizer to be used for policy/actor,
```

Comment:
torch.optim is a package, not a type.

Reply:
please correct this error throughout the docstring. it's present in several places (replay buffer, q_function, etc.)